src/applications/appeals/10182/containers/IntroductionPage.jsx

```diff
@@ -12,67 +22,116 @@ class IntroductionPage extends React.Component {
 
   render() {
     const { formConfig, pageList } = this.props.route;
+    const { formId, prefillEnabled, savedFormMessages } = formConfig;
+    const sipOptions = {
+      useActionLinks: true,
+      hideUnauthedStartLink: true,
+      formId,
+      prefillEnabled,
+      pageList,
+      messages: savedFormMessages,
+      startText: 'Start the Board Appeal request',
+    };
```

Comment:
I like this pattern!

Reply:
`useActionLinks` may not be necessary if the decision to make the save-in-progress start into an action link - see https://github.com/department-of-veterans-affairs/vets-website/pull/16752